### ISSUE TITLE
Recurring event shows up with wrong date in Simple-Calendar (v4.2.1)

### ISSUE THREAD

**Issue by schneis** [SimpleMobileTools/Simple-Calendar#662]
Background - my phone & tablet both sync my Exchange server mailbox to the device.  I had been using Google Calendar but recently switched both devices to using Simple-Calendar as a trial.

The recurring appointment on my calendar (every 4 weeks effective 2/28/2013) to take out my garbage is showing one week later than it should with Simple-Calendar.  My desktop running Outlook, and Google Calendar on both devices show the correct date.

For October, the correct date of the event is Thursday, October 4th.  Simple-Calendar is showing the date as Thursday, October 11th.

What data should I try to collect to help resolve this bug?

**Comment by youdly** [comment]
Can you export the reminder to a file? Or try to email the reminder to another email and download the attachment.
I have a similar 4 weeks reminder that looks like this:
```
BEGIN:VCALENDAR
BEGIN:VEVENT
DTSTART;VALUE=DATE:20181015
DTEND;VALUE=DATE:20181016
UID:topup_card
STATUS:CONFIRMED
RRULE:FREQ=WEEKLY;INTERVAL=4;UNTIL=20191231T000000Z
SUMMARY:Topup Card
END:VEVENT
END:VCALENDAR
```

**Comment by schneis** [comment]
Simple Calendar does not make exporting a specific event very easy.  This should be the event:

```
BEGIN:VCALENDAR
PRODID:-//Simple Mobile Tools//NONSGML Event Calendar//EN
VERSION:2.0
BEGIN:VEVENT
SUMMARY:Garbage Pick Up
DESCRIPTION:
\n
UID:Caldav-16-354
LAST-MODIFIED:19700101T000000Z
DTSTART:20130228T140000Z
DTEND:20130228T140000Z
STATUS:CONFIRMED
RRULE:FREQ=WEEKLY;INTERVAL=4;BYDAY=TH
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-P0DT8H0M0S
END:VALARM
EXDATE:20181101}
END:VEVENT
END:VCALENDAR

```

**Comment by youdly** [comment]
Recreate the reminder (with the same start date but slightly different name) through the app and see if this issue recurs. Export both events and compare them.

**Comment by schneis** [comment]
I created a new event in Outlook with the same settings.  It shows up with the correct recurrence in both Outlook on Windows and Google Calendar on Android, but shows up 1 week off again using Simple-Calendar.  I doubt that the issue is on the server side - this appears to be a bug in Simple-Calendar when parsing a long-running recurring event.

```
BEGIN:VCALENDAR
PRODID:-//Simple Mobile Tools//NONSGML Event Calendar//EN
VERSION:2.0
BEGIN:VEVENT
SUMMARY:Garbage Test
DESCRIPTION:
\n
UID:Caldav-16-392
LAST-MODIFIED:19700101T000000Z
DTSTART:20130228T140000Z
DTEND:20130228T140000Z
STATUS:CONFIRMED
RRULE:FREQ=WEEKLY;INTERVAL=4;BYDAY=TH
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-P0DT8H0M0S
END:VALARM
END:VEVENT
END:VCALENDAR

```

**Comment by youdly** [comment]
I meant create a new reminder using Simple-Calendar instead.

**Comment by tibbi** [comment]
recheck with the pro version 6.0 released soon please

**Comment by schneis** [comment]
I would have appreciated you keeping this open until you had confirmation that this was resolved.  The bug is still present in Pro v6.0!

Export of event created with Pro v6.0 which still shows on the wrong week (yet Outlook, Google Calendar, etc show the event on the correct date):
```
BEGIN:VCALENDAR
PRODID:-//Simple Mobile Tools//NONSGML Event Calendar//EN
VERSION:2.0
BEGIN:VEVENT
SUMMARY:Test
UID:0
LAST-MODIFIED:20181117T095433Z
DTSTART:20130228T140000Z
DTEND:20130228T140000Z
STATUS:CONFIRMED
RRULE:FREQ=WEEKLY;INTERVAL=4;BYDAY=TH
BEGIN:VALARM
ACTION:DISPLAY
TRIGGER:-P1DT0H0M0S
END:VALARM
END:VEVENT
END:VCALENDAR

```

As there doesn't seem to be any interest in actually addressing this issue (based on closing the bug and claiming it has been fixed in a pay version when there has been no change), I don't see the point in being run in circles to provide more information.  I'll be uninstalling this suite of apps and going elsewhere.

**Comment by tibbi** [comment]
this glitch related to repetition by X weeks should be fixed in 6.0.2 which will be released soon, reopen if not
